README.md

```diff
@@ -1,31 +1,32 @@
 # Rust RFCs
-[Rust RFCs]: #rust-rfcs
+[Opening]: #rust-rfcs
```

Comment:
That one didn't even have one before I added it; I was trying to keep the TOC unchanged.